packages/pwa-kit-react-sdk/scripts/create-hash-manifest.js

```diff
@@ -17,6 +17,10 @@ const currentDate = Date.now()
 const APP_MANIFEST_NAME = 'cache-hash-manifest.json'
 const LOADER_MANIFEST_NAME = 'loader-cache-hash-manifest.json'
 
+console.warn(
+    'Deprecated. The create-hash-manifest script will be removed in our next major version.'
```

Comment:
@vcua-mobify Can you please name the specific major version you are talking about here and follow it up with a ticket in GUS? If we just say "the next version", nobody knows when they have to act and we won't know that we have a deprecation plan to follow.

Reply:
@vcua-mobify FYI - that I added Bug P3, label which auto-created work item in GUS. I agree with the comment from Oliver. I guess that takes care of my 'let's park it' comment, I just didn't wanna lose the action item in the future major version.

Reply:
PR has been updated to specify create-hash-manifest's removal in pwa-kit-react-sdk v2.0.0.